doc/xds-test-descriptions.md

```diff
@@ -779,4 +779,50 @@ that during some 10-second interval, all traffic goes to the other four
 backends and all requests end with the `OK` status.
 3.  The test driver removes the client configuration to send metadata. The
 driver asserts that during some 10-second interval, traffic is equally
-distributed among the five backends, and all requests end with the `OK` status.
\ No newline at end of file
+distributed among the five backends, and all requests end with the `OK` status.
+
+### custom_lb
+This test verifies that a custom load balancer policy can be configured in the
+client. It also verifies that when given a list of policies the client can
+ignore a bad one and try the next one on the list until it finds a good one.
+
+Client parameters:
+
+1.  --num_channels=1
+2.  --qps=100
+
+Load balancer configuration:
+
+One MIG with a single backend.
+
+The `backendService` will have the following `localityLbPolicies` entry:
```

Comment:
I am not sure this is portable. Maybe this should be reworded, e.g. mention that `test.RpcBehaviorLoadBalancer` custom LB policy should be defined and understand the config and `test.ThisLoadBalancerDoesNotExist` should not be defined.

I wonder if this should pass `--service_config_json` to client and said config should configure the policies.

Reply:
I added language that explains what the LB should do. For reference you can look at the Java implementation here: https://github.com/grpc/grpc-java/blob/master/interop-testing/src/main/java/io/grpc/testing/integration/RpcBehaviorLoadBalancerProvider.java

We can't use service config directly like this because the idea is to test things end to end with Traffic Director. The test will set up the GCP backendService with this custom LB config and TD passes it down to the client that will read it and set up load balancing accordingly. The test itself is here: https://github.com/grpc/grpc/blob/master/tools/run_tests/xds_k8s_test_driver/tests/custom_lb_test.py